<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>notify</class>
 <widget class="QDialog" name="notify">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>231</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="button_confirm">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>confirm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_back">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>200</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_send">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>200</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>send</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>68</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Summary</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
